src/components/containers/visibility.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import styled from 'styled-components'
+import device, { size } from 'themes/device'
+import { useBrowserResize } from 'components/hooks/use-browser-resize'
+
+type ResponsiveContainerProps = {
+    children: ReactElement
+}
+
+const DesktopLayer = styled.div`
+    @media ${device.tablet} {
+        display: none;
+    }
+`
+const MobileLayer = styled.div`
+    @media ${device.mobile} {
+        display: none;
+    }
+`
```

Comment:
Might be better to use the bp from `size` instead of `device` since we want to control min-width and also max-width. We can accept the props of `size` and then pass it in here and also the `useBrowserResize`. Rest looks good 👍